[2024-04-09T16:28:59.086+0000] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: 21120051.2112 scheduled__2024-04-09T16:23:50.277723+00:00 [queued]>
[2024-04-09T16:28:59.101+0000] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: 21120051.2112 scheduled__2024-04-09T16:23:50.277723+00:00 [queued]>
[2024-04-09T16:28:59.102+0000] {taskinstance.py:2193} INFO - Starting attempt 1 of 2
[2024-04-09T16:28:59.332+0000] {taskinstance.py:2214} INFO - Executing <Task(PythonOperator): 2112> on 2024-04-09 16:23:50.277723+00:00
[2024-04-09T16:28:59.341+0000] {standard_task_runner.py:60} INFO - Started process 1414 to run task
[2024-04-09T16:28:59.348+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', '21120051', '2112', 'scheduled__2024-04-09T16:23:50.277723+00:00', '--job-id', '133', '--raw', '--subdir', 'DAGS_FOLDER/pipeline.py', '--cfg-path', '/tmp/tmpjg8g3eqs']
[2024-04-09T16:28:59.353+0000] {standard_task_runner.py:88} INFO - Job 133: Subtask 2112
[2024-04-09T16:28:59.419+0000] {task_command.py:423} INFO - Running <TaskInstance: 21120051.2112 scheduled__2024-04-09T16:23:50.277723+00:00 [running]> on host dcb7ab325c84
[2024-04-09T16:28:59.531+0000] {taskinstance.py:2510} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='21120051' AIRFLOW_CTX_DAG_ID='21120051' AIRFLOW_CTX_TASK_ID='2112' AIRFLOW_CTX_EXECUTION_DATE='2024-04-09T16:23:50.277723+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-04-09T16:23:50.277723+00:00'
[2024-04-09T16:28:59.617+0000] {logging_mixin.py:188} INFO - Collection already exists
[2024-04-09T16:28:59.618+0000] {python.py:202} INFO - Done. Returned value was: {'status': 'success', 'collection': '21120051', 'vectorParams': "{'size': 1536, 'distance': <Distance.COSINE: 'Cosine'>}"}
[2024-04-09T16:28:59.648+0000] {taskinstance.py:1149} INFO - Marking task as SUCCESS. dag_id=21120051, task_id=2112, execution_date=20240409T162350, start_date=20240409T162859, end_date=20240409T162859
[2024-04-09T16:28:59.678+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2024-04-09T16:28:59.710+0000] {taskinstance.py:3309} INFO - 1 downstream tasks scheduled from follow-on schedule check
